app/screens/integration_selector/index.tsx

```diff
@@ -457,19 +456,19 @@ function IntegrationSelector(
         switch (dataSource) {
             case ViewConstants.DATA_SOURCE_USERS:
                 text = {
-                    id: intl.formatMessage({id: 'mobile.integration_selector.loading_users'}),
+                    id: 'mobile.integration_selector.loading_users',
                     defaultMessage: 'Loading Users...',
                 };
                 break;
             case ViewConstants.DATA_SOURCE_CHANNELS:
                 text = {
-                    id: intl.formatMessage({id: 'mobile.integration_selector.loading_channels'}),
+                    id: 'mobile.integration_selector.loading_channels',
```

Comment:
these id's needs to be wrapped using the `t` function found in the `@i18n` module.

After that, Make sure to run `npm run i18n-extract` to make sure all strings are present